Formula/l/llamafile.rb

```diff
@@ -0,0 +1,43 @@
+class Llamafile < Formula
+  desc "Distribute and run LLMs with a single file"
+  homepage "https://github.com/Mozilla-Ocho/llamafile"
+  url "https://github.com/Mozilla-Ocho/llamafile/archive/refs/tags/0.6.tar.gz"
+  sha256 "9ac8449268287c45fb015f7068b223be7336c6d7f8170f3adb9a370b65e8b9f0"
+  license "Apache-2.0"
+  head "https://github.com/Mozilla-Ocho/llamafile.git", branch: "main"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "make"
+
+  skip_clean "bin/llamafile",
+             "bin/llamafile-convert",
+             "bin/llamafile-perplexity",
+             "bin/llamafile-quantize",
+             "bin/llava-quantize",
+             "bin/zipalign"
+
+  def install
+    # Call `make` as `gmake` to use GNU `make`
+    system "gmake", "-j8"
+    system "gmake", "install", "PREFIX=#{prefix}"
```

Comment:
I doubt macOS ships with a GNU make

Reply:
AFAIK it doesn't. [`depends_on "make"`](https://github.com/Homebrew/homebrew-core/pull/160198/files#diff-fc976828e2e8390e61d7bdae4bafcd83d394de7d25f6d82ce45e6cf95e1b8539R9) should require it, right?